closed/OpenJ9.gmk

```diff
@@ -315,6 +322,11 @@ endif
 ifneq (,$(OPENJ9_DEVELOPER_DIR))
   CUSTOM_COMPILER_ENV_VARS += DEVELOPER_DIR="$(OPENJ9_DEVELOPER_DIR)"
 endif
+ifeq (true,$(OPENJ9_ENABLE_INLINETYPES))
+  ifneq (true,$(OPENJ9_ENABLE_CMAKE))
+    CUSTOM_COMPILER_ENV_VARS += J9VM_OPT_VALHALLA_VALUE_TYPES=1
```

Comment:
The environment is not the right way to communicate this to cmake; we should be using `-D` command-line options like [1]. I'm not sure using cmake in this version is fully supported yet, so we may not need to worry too much about this now.

[1] https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/openj9/closed/OpenJ9.gmk#L378-L382